<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AoideTrackSearchOverlayFilterDlg</class>
 <widget class="QDialog" name="AoideTrackSearchOverlayFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>aoide Track Search Overlay Filter</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>448</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBpm">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMinBpm">
        <item>
         <widget class="QLabel" name="minBpmLabel">
          <property name="text">
           <string>Min. BPM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="minBpmDoubleSpinBox">
          <property name="maximumSize">
           <size>
            <width>147</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minBpmResetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutMaxBpm">
        <item>
         <widget class="QLabel" name="maxBpmLabel">
          <property name="text">
           <string>Max. BPM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="maxBpmDoubleSpinBox">
          <property name="maximumSize">
           <size>
            <width>147</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="maxBpmResetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutGenre">
      <item>
       <widget class="QLabel" name="genreTextLabel">
        <property name="text">
         <string>Genres (any, split at ';')</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="genreTextLineEdit">
        <property name="maximumSize">
         <size>
          <width>183</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutComment">
      <item>
       <widget class="QLabel" name="commentTextLabel">
        <property name="text">
         <string>Comment terms (all, split at whitespace)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commentTextLineEdit">
        <property name="maximumSize">
         <size>
          <width>183</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AoideTrackSearchOverlayFilterDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AoideTrackSearchOverlayFilterDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
